homeassistant/const.py

```diff
@@ -185,6 +185,7 @@
 DEVICE_CLASS_TIMESTAMP = 'timestamp'
 DEVICE_CLASS_PRESSURE = 'pressure'
 DEVICE_CLASS_POWER = 'power'
+DEVICE_CLASS_SWITCH = 'switch'
```

Comment:
If adding a new sensor device class, this should be [documented](https://www.home-assistant.io/components/sensor/#device-class) - there might be implications outside this component, too.